app/controllers/api/feedback_submissions_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  class FeedbackSubmissionsController < ApiController
+    def create
+      @feedback = FeedbackSubmission.new(feedback_params)
+
+      if @feedback.save
+        ZendeskTicketsJob.perform_later(@feedback)
+      else
+        render status: 422, json: { messages: @feedback.errors.full_messages }
```

Comment:
What do you mean by 'error state' in your second last sentence? Do you mean something different than a 422?

I don't see it as a bad thing to return validations errors on create, mainly for 2 reasons:
- Some errors will happen on creation even after pre-validating, for example unique validations that are prone to race conditions.
- Instead of having 2 requests for creation (validation + create) we only need 1 request (If I understand your suggested pattern correctly). Less requests are better imo.

Good to know that you are following some form the Github's API, I didn't really know what to do with the errors. What do you think of trying to encapsulate this design decisions on a ActiveModel Serializer adapter, or writing them somewhere, perhaps on a wiki page on the project?

Out of curiosity, did you consider some API standards like json-api?